tools/library/src/singleton.cu

```diff
@@ -54,10 +50,8 @@ Singleton::Singleton() {
 }
 
 Singleton const & Singleton::get() {
-  if (!instance.get()) {
-    instance.reset(new Singleton);
-  }
-  return *instance.get();
+  static Singleton instance;
+  return &instance;
```

Comment:
`error: no suitable constructor exists to convert from "cutlass::library::Singleton *" to "cutlass::library::Singleton"`

Reply:
sorry this is a mistake, I will fix it